docker/tpl-ubuntu-clang.Dockerfile

```
@@ -73,7 +73,7 @@ RUN apt-get install -y --no-install-recommends \
 RUN --mount=src=.,dst=$SRC_DIR,readwrite cd ${SRC_DIR} && \
      mkdir -p ${GEOSX_TPL_DIR} && \
      ./scripts/uberenv/uberenv.py \
-       --spec "%clang@${CLANG_MAJOR_VERSION} ~openmp+docs ^caliper~gotcha~sampler~libunwind~libdw~papi" \
+       --spec "%clang@${CLANG_MAJOR_VERSION} ~openmp+docs ^caliper~gotcha~sampler~libunwind~libdw~papi~shared" \
```

```suggestion
       --spec "%clang@${CLANG_MAJOR_VERSION} ~shared~openmp+docs ^caliper~gotcha~sampler~libunwind~libdw~papi" \
```
This will apply `~shared` to geos, as-is caliper will be built static.